code/modules/reagents/Chemistry-Reagents/Pyrotechnic-Reagents.dm

```diff
@@ -227,9 +227,11 @@
 			if (reac_volume >= 5)
 				M.bodytemperature = max(M.bodytemperature - 200, TCMB)
 				M.adjust_fire_stacks(-(3*reac_volume))
-				M.adjustStaminaLoss(20*reac_volume)
-				M.adjustFireLoss(0.25*reac_volume)
-				M.adjustOxyLoss(0.75*reac_volume)
+				M.adjustFireLoss(0.125*reac_volume) //Sorry for snowflakey numbers~
+				M.adjustOxyLoss(0.375*reac_volume)
+				var/brutedmg = M.getBruteLoss()
+				M.adjustBruteLoss(brutedmg*0.5)
```

Comment:
You could just have M.adjustBruteLoss(0.5*M.getBruteLoss()) instead of creating a snowflake var.